src/main/scala/sbttravisci/TravisCiPlugin.scala

```diff
@@ -26,34 +26,50 @@ object TravisCiPlugin extends AutoPlugin {
 
     // parses Scala versions out of .travis.yml (doesn't support build matrices)
     crossScalaVersions := {
-      import scala.collection.JavaConverters._
-      Using.fileInputStream(baseDirectory.value / ".travis.yml") { fis =>
-        val yaml = Option(new org.yaml.snakeyaml.Yaml().load(fis))
-          .collect { case map: java.util.Map[_, _] => map }
-
-        def fromRoot = yaml
-          .flatMap(map => Option(map get "scala"))
-          .collect {
-            case versions: java.util.List[_] => versions.asScala.toList map (_.toString)
-            case version: String             => version :: Nil
-          }
-          .getOrElse(Nil)
-
-        def fromMatrixInclude = yaml
-          .flatMap(map => Option(map get "matrix"))
-          .collect { case map: java.util.Map[_, _] => Option(map get "include") }.flatten
-          .collect { case versions: java.util.List[_] =>
-            versions.asScala.toList.collect { case map: java.util.Map[_, _] =>
-              Option(map get "scala") map (_.toString)
-            }.flatten
+      val log = sLog.value
+
+      val manifest = baseDirectory.value / ".travis.yml"
+      val default = Seq("2.10.6")     // this avoids cyclic dependency issues
```

Comment:
It's not technically correct. And it might not even work (because of the way buildSettings are processed or because of derive or because of both). But it's probably a better defailt.